### ISSUE TITLE
[$250] Expense-Delete and replace options displayed for receipt in archived workspace offline

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#50558]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___
 
**Version Number:** 9.0.47-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Issue was found when executing this PR:** https://github.com/Expensify/App/pull/50395
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Open the app and log in
2. Navigate to a workspace chat
3. Disable internet connection
3. Create an expense with a receipt
4. Navigate to the list of workspaces and delete the workspace
5. Open the archived workspace chat
6. Select the expense > tap 3 dots

## Expected Result:
The "Delete" and "Replace" options should not be present

## Actual Result:
The "Delete" and "Replace" options are displayed when the workspace is deleted offline

## Workaround:
Unknown

## Platforms:

- [x] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/ab81d261-f065-4c24-807a-176a8fecba71

 
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021884218250507093585</li>
        <li>Upwork Job ID: 1884218250507093585</li>
        <li>Last Price Increase: 2025-02-04</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @mananjadhav</details>

**Comment by IuliiaHerets** [comment]
We think that this bug might be related to #wave-collect - Release 1

**Comment by IuliiaHerets** [comment]
@dylanexpensify  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by dylanexpensify** [comment]
ongoing

**Comment by dylanexpensify** [comment]
Hey @johncschuster! I'm heading out on parental leave so reassigning this! TY! 🙇‍♂️

**Comment by johncschuster** [comment]
@Krishna2323 how's this one going?

**Comment by johncschuster** [comment]
@Krishna2323 how's this one going? Were you able to get a PR up for this?

**Comment by johncschuster** [comment]
Thanks for your comment, @Krishna2323! Let me review this with fresh eyes in the morning.

**Comment by johncschuster** [comment]
I didn't get to this one today. I will return to [this comment](https://github.com/Expensify/App/issues/50558#issuecomment-2439783858) tomorrow.

**Comment by johncschuster** [comment]
No update today. Still catching up from a shorter week last week.

**Comment by johncschuster** [comment]
Ok! I'm back. @Krishna2323 I'm not quite sure if it's intended to delete and replace the receipt image for an archived expense. I'll need to check on that on Monday.

**Comment by johncschuster** [comment]
I didn't get to dig into this today. I should have some more time tomorrow!

**Comment by johncschuster** [comment]
No update on this one. I'll need to push this one forward tomorrow.

**Comment by johncschuster** [comment]
I'm working a shorter week this week and couldn't fit this in today

**Comment by johncschuster** [comment]
I'm catching up from being OOO for the holiday last week. Will get this slotted in tomorrow.

**Comment by johncschuster** [comment]
I couldn't action this today. I will check it out this weekend.

**Comment by johncschuster** [comment]
No update. I'll bring this to Slack tomorrow for discussion on the intended behavior.

**Comment by johncschuster** [comment]
I didn't get to this today. I will write up the discussion over the weekend and post it Monday

**Comment by johncschuster** [comment]
No update

**Comment by garrettmknight** [comment]
@johncschuster should this be `Internal` or `External`?

**Comment by johncschuster** [comment]
@garrettmknight, I'm not certain. @Krishna2323, can you help us determine if this issue should be `Internal` or `External`?

**Comment by johncschuster** [comment]
@Krishna2323 bump!

**Comment by garrettmknight** [comment]
Let's start external if we don't know.

**Comment by mananjadhav** [comment]
Just got assigned and it has 73 conversations. Will review the previous comments soon.

**Comment by FitseTLT** [comment]
This should be expected. cc @srikarparsi

**Comment by srikarparsi** [comment]
Yes, I agree I think this is expected now that we are not archiving expense reports. When we take these actions [online](https://github.com/Expensify/App/issues/50558#issue-2578608890), the replace button also exists. cc @trjExpensify to confirm

**Comment by trjExpensify** [comment]
Yeah, when in doubt, check OldDot. You can edit an expense on a deleted workspace. We're in the process of adding `Move expense` and `Change workspace` actions soon, which will allow you to move the expense or expense report to another workspace as well. :)

**Comment by johncschuster** [comment]
Thanks @trjExpensify and @srikarparsi! In that case, is this issue even necessary? It sounds like the change in behavior makes this issue irrelevant.

**Comment by mananjadhav** [comment]
@trjExpensify @srikarparsi Should we close this one out?

**Comment by trjExpensify** [comment]
Yup!
